src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/CleanupBroker.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NServiceBus
+{
+    using NUnit.Framework;
+    using static Bullseye.Targets;
+
+    [SetUpFixture]
+    public class BrokerCleanup
+    {
+        [OneTimeSetUp]
+        public void RunBeforeAnyTests()
+        {
+            Target("delete-virtual-host", () => Broker.DeleteVirtualHost());
```

Comment:
I'm curious what Bullseye is doing that can't be accomplished by:

```
Broker.DeleteVirtualHost();
Broker.CreateVirtualHost();
Broker.AddUserToVirtualHost();
```

Reply:
You should be able to remove the Bullseye reference for this moved code.